docs/src/integrate/nodejs-api.md

```diff
@@ -295,6 +300,21 @@ This method loads a formatter. Formatters convert lint results to a human- or ma
 * (`Promise<LoadedFormatter>`)<br>
   The promise that will be fulfilled with a [LoadedFormatter] object.
 
+### ◆ eslint.hasFlag()
+
+This method is used to determine if a given feature flag is set, as in this example:
+
+```js
+if (eslint.hasFlag("x_feature")) {
+    // handle flag
+}
+```
+
+#### Parameters
+
+* `flagName` (`string`)<br>
+  The flag to check.
+
```

Comment:
Shall we add a section `#### Return Value` like for the other methods?

Reply:
Good catch.